src/html/index.html

```diff
@@ -54,11 +54,11 @@
     <link rel="icon" href="{{{ cdn }}}/images/favicon.png" type="image/png" sizes="196x196" />
   </head>
   <body data-decorator="load-ga display-rp-name prevent-clickjack">
-    <!--
-    <div class="banner">
-      <p></p>
+    <div data-decorator="check-if-maintenance-mode" hidden>
+      <div class="banner">
+        <p>We are currently experiencing an issue & working hard to resolve it. Please try to login again in a few minutes.  Request help via <a href="mailto:moc@mozilla.com">moc@mozilla.com</a>, IRC #moc or Slack #mozsm-all.</p>
```

Comment:
`s/&/&amp;/ `

note, were we cleared to use `#mozsm-all` vs `#servicedesk`? the other page has `#servicedesk`